packages/nx/src/command-line/connect/connect-to-nx-cloud.ts

```diff
@@ -8,16 +8,29 @@ import {
   isNxCloudUsed,
 } from '../../utils/nx-cloud-utils';
 import { runNxSync } from '../../utils/child-process';
+import { NxJsonConfiguration } from '../../config/nx-json';
+
+export function onlyDefaultRunnerIsUsed(nxJson: NxJsonConfiguration) {
+  if (!nxJson.tasksRunnerOptions) {
+    if (!nxJson.nxCloudAccessToken) return false;
+    return true;
+  }
+  const defaultRunner = nxJson.tasksRunnerOptions?.default;
+  if (
+    !defaultRunner.runner ||
+    defaultRunner.runner === 'nx/tasks-runners/default'
+  ) {
+    return true;
+  }
+  return false;
```

Comment:
This is missing logic to check if they ONLY have the default configured.

EVERY task runner options value should be the default task runner.

Reply:
Jason and I talked about this one offline, cloud only cares about the default runner anyways.